components/movementsensor/movementsensor.go

```diff
@@ -43,10 +43,25 @@ func init() {
 		},
 	})
 
-	registerCollector("GetPosition", func(ctx context.Context, ms MovementSensor) (interface{}, error) {
+	registerCollector("Position", func(ctx context.Context, ms MovementSensor) (interface{}, error) {
 		p, _, err := ms.Position(ctx)
```

Comment:
@AaronCasas Since Position returns a *Point struct, triple checking right now that it's properly being handled in InterfaceToMap.

Reply:
> doesn't seem to work if the fields are unexported

Yup this is right - only exported fields can be parsed